.circleci/config.yml

```diff
@@ -1,47 +1,45 @@
 version: 2
 jobs:
   build:
+    working_directory: /content-collection-unfolder
     docker:
       - image: golang:1
-    working_directory: /go/src/github.com/Financial-Times/content-collection-unfolder
     environment:
       GOPATH: /go
       CIRCLE_TEST_REPORTS: /tmp/test-results
       CIRCLE_COVERAGE_REPORT: /tmp/coverage-results
-
     steps:
       - checkout
       - run:
           name: External Dependencies
           command: |
-            go get -u github.com/mattn/goveralls
-            go get -u github.com/jstemmer/go-junit-report
-            go get -u github.com/haya14busa/goverage
-            go get -u github.com/golang/dep/cmd/dep
+            GO111MODULE=off go get github.com/mattn/goveralls
+            GO111MODULE=off go get github.com/jstemmer/go-junit-report
+            curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.21.0
+            git clone git@github.com:Financial-Times/upp-coding-standard.git
```

Comment:
There is a release in this repo, you can use `wget https://raw.githubusercontent.com/Financial-Times/upp-coding-standard/v1.0.0/golangci-config/.golangci.yml` instead of cloning the repo.